SCF-Assignment-2/Question 1/ConversionClass.java

```diff
@@ -0,0 +1,40 @@
+package Hexadecimal;
+
+/*
+ * this class includes function which will convert hexadecimal number to decimal number and vice versa.
+ */
+public class ConversionClass {
+	String hexCharacter = "0123456789ABCDEF";

```

Comment:
If no access specifier is specified the field becomes package private which means this field is accessible to all the classes in the same package. Thus, we need to specify these as private explicitly.